Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityProfileCloneWindow.cs

```diff
@@ -37,8 +37,9 @@ public SubProfileAction(ProfileCloneBehavior behavior, SerializedProperty proper
             public System.Type ProfileType;
         }
 
+        private const string DefaultCustomProfileFolder = "Assets/MixedRealityToolkit.Generated/CustomProfiles";
         private const string IsCustomProfileProperty = "isCustomProfile";
-        private static readonly Vector2 MinWindowSizeBasic = new Vector2(500, 140);
+        private static readonly Vector2 MinWindowSizeBasic = new Vector2(500, 170);
```

Comment:
Why the change from 140 -> 170? 

By the way, these both feel like "magic numbers" that might warrent a named constant for clarity.

Reply:
Maybe it's to make room for the new HelpBox on line 193 (shown below)?

Reply:
Yep, it's a height thing because the new info message increases the required height of the window. I thought about commenting this (i.e. 'this determines the size of the window') but then didn't do it because the naming of the window is relatively obvious and missing changing this leads to a very obvious side effect.